utils/gen_caffe2_model.py

```diff
@@ -0,0 +1,54 @@
+## This is a helper script that generates simple Caffe2 models.
```

Comment:
Could you add a comment showing what the invocation of the script looks like?
I.e., what parameters you need to pass.